packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -478,6 +507,76 @@ class AndroidCameraCameraX extends CameraPlatform {
         exposureState.exposureCompensationStep;
   }
 
+  /// Sets the focus mode for taking pictures.
+  ///
+  /// Setting [FocusMode.locked] will lock current focus point if one exists or
+  /// center of entire sensor area if not and will stay locked until either:
+  ///   * Another focus point is set via [setFocusPoint] (which will then become
+  ///     the locked focus point), or
+  ///   * Locked focus mode is unset by setting [FocusMode.auto].
+  @override
+  Future<void> setFocusMode(int cameraId, FocusMode mode) async {
+    switch (mode) {
+      case FocusMode.auto:
+        if (_currentFocusMode == FocusMode.auto) {
+          // CameraX uses auto-focus mode by default, so no need to reconfigure
```

Comment:
Yes upon further thought I think this is true. Will update. And the other comment re:getting out of state shouldn't be a problem since the `FocusMeteringResult` will tell us if the action was successful or not.